ehsm_kms_service/router.js

```diff
@@ -25,6 +25,7 @@ const {
 } = require('./key_management_apis')
 const {
   createSecret,
+  updateSecret

```

Comment:
so why rename the api to updatesecret?

Reply:
The name will be change to UpdateSecretDesc